src/mscorlib/mscorlib.csproj

```diff
@@ -181,7 +181,8 @@
   </PropertyGroup>
 
   <Import Project="$(MSBuildThisFileDirectory)Tools\Versioning\GenerateVersionInfo.targets"/>
-  <Import Project="GenerateSplitStringResources.targets"/>
+  <Import Project="GenerateSplitStringResources.Windows.targets" Condition="'$(OsEnvironment)'=='Windows_NT'"/>
```

Comment:
My suggestion was that there should only ever be a single copy of the file  that will be used both for open and closed builds. The only difference would be to use resgen for closed build and the python script for open build.

This will ensure that the overall file can be consistent for both the builds. It sounds fine to look into doing this merge once the change goes in so that we unblock the open build work. 

Thanks @ellismg.